src/api/edge/imagesInfo.js

```diff
@@ -5,7 +5,8 @@ export const useGetImageSet = () => {
   const axios = useAxiosWithPlatformInterceptors();
   const query = { limit: 10, offset: 0, sort_by: '-created_at' };
   const params = getTableParams(query);
-  return (id) => axios.get(`${EDGE_API}/image-sets/${id}?${params}`);
+  return (p) =>
```

Comment:
This is still getting passed an `id` or? 

In general we should avoid using one character variable names and try to be more descriptive.